test/tests/parallel/set3/mta/privacy.bats

```diff
@@ -25,7 +25,7 @@ function teardown_file() { _default_teardown ; }
 
 # this test covers https://github.com/docker-mailserver/docker-mailserver/issues/681
 @test "(Postfix) remove privacy details of the sender" {
-  _run_in_container_bash "openssl s_client -quiet -starttls smtp -connect 0.0.0.0:587 < /tmp/docker-mailserver-test/email-templates/send-privacy-email.txt"
+  _send_email --port 587 -tls --auth LOGIN --auth-user user1@localhost.localdomain --auth-password mypassword 'privacy'
```

Comment:
```suggestion
  _send_email \
    --port 587 -tls --auth LOGIN \
    --auth-user user1@localhost.localdomain \
    --auth-password mypassword \
    'privacy'
```

Reply:
Context:
- `DATA` for privacy file contains a `User-Agent` header being tested for removal on relay (_presently sends from self to self_).